prime-router/src/main/kotlin/Receiver.kt

```diff
@@ -15,8 +15,11 @@ import java.time.ZoneId
  * @param topic defines the set of schemas that can translate to each other
  * @param customerStatus defines if the receiver is fully onboarded
  * @param translation configuration to translate
- * @param jurisdictionalFilter defines the set of elements and regexes that filter the data for this receiver
- * @param qualityFilter defines the set of elements and regexes that do qualiyty filtering on the data for this receiver
+ * @param jurisdictionalFilter defines the geographic region filters for this receiver
+ * @param qualityFilter defines the filters that remove data, based on quality criteria
+ * @param routingFilter defines the filters that remove data the receiver does not want, based on who sent it.
```

Comment:
```suggestion
 * @param routingFilter The original use case was to use this to defines the filters that remove data the receiver does not want, based on who sent it.  However, its available for any general purpose use.
```

Good thought, thank you.  @carlosfelix2 suggested changing the name from routingFilter, but I haven't come up with a good name yet (uh, `otherFilter` ?), so  I'm going to leave it be for now.